src/aca_main.cpp

```diff
@@ -150,6 +157,15 @@ int main(int argc, char *argv[])
     case 'p':
       g_rpc_protocol = optarg;
       break;
+    case 'c':
```

Comment:
Let us also update our development instruction https://github.com/futurewei-cloud/alcor-control-agent/blob/master/src/README.md on how to run ACA with those new parameters.